railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_1.rb.tt

```diff
@@ -15,8 +15,9 @@
 #
 # This will reduce the size of the load path, making `require` faster if you don't use bootsnap, or reduce the size
 # of the bootsnap cache if you use it.
-#++
-# Rails.application.config.add_autoload_paths_to_load_path = false
+#
+# To set this configuration, add the following line to `config/application.rb`:
```

Comment:
I see that we wrote an emphatic "NOT this file" for a similar case:

https://github.com/rails/rails/blob/4626b71413612e2c58d88d0d89da649c969fdb50/railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_1.rb.tt#L239-L241

Should we do that here?  On one hand, if someone blindly uncomments the config, it will at least raise an error.  On the other hand, I can envision someone "handling" that error by adding `Rails.application.`.  Perhaps they might even submit a PR to "fix" it. :smile:

Reply:
Sounds good to me, updated!